examples/quantum_die.py

```diff
@@ -28,47 +28,43 @@ def qubits_needed(n):
     """
     return int(math.ceil(math.log(n, 2)))
 
+def qvm(n):
+    """
+    Connect to a QVM that simulates n sides.
+    """
+    if qubits_needed(n) == 9:
```

Comment:
What is the benefit gained by using `9q-square-qvm`? Could we make this simpler for example purposes and always follow the same path here, without special casing?